<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCR</class>
 <widget class="QMainWindow" name="SCR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>787</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="chart"/>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_3"/>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignLeft">
           <widget class="QSpinBox" name="spinBox"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Algorytm:</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <widget class="QRadioButton" name="checkBox_PRIORYTET">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>30</y>
               <width>82</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>prio</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="checkBox_RMS">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>60</y>
               <width>82</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>rms</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="checkBox_EDF">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>90</y>
               <width>82</width>
               <height>17</height>
              </rect>
             </property>
             <property name="text">
              <string>edf</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Rysuj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Koniec</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="tabele">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>254</height>
           </size>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <column>
           <property name="text">
            <string>Czas przetwarzania:</string>
           </property>
           <property name="textAlignment">
            <set>AlignJustify|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Okres (Termin):</string>
           </property>
           <property name="textAlignment">
            <set>AlignJustify|AlignVCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Priorytet</string>
           </property>
           <property name="textAlignment">
            <set>AlignJustify|AlignVCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
